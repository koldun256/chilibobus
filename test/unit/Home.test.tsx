import { render } from "@testing-library/react";
import React from "react";
import { Home } from "../../src/client/pages/Home";

test("Главная страница отображает нужную информацию", () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot("Главная страница");
});
