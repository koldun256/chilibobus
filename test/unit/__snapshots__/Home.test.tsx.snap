// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Главная страница отображает нужную информацию: Главная страница 1`] = `
<div>
  <div
    class="Home"
  >
    <div
      class="row"
    >
      <div
        class="col bg-orange text-white py-4 bg-opacity-75"
      >
        <p
          class="display-3"
        >
          Welcome to Kogtetochka store!
        </p>
        <p
          class="lead"
        >
          We have a large assortment of scratching posts!
        </p>
      </div>
    </div>
    <div
      class="row mb-4"
    >
      <div
        class="col-12 col-md-4 bg-light py-3"
      >
        <h1>
          Stability
        </h1>
        <p
          class="lead"
        >
          Our scratching posts are crafted with precision and designed for unparalleled stability. Made from high-quality materials, they provide a sturdy platform for your cat's scratching needs.
        </p>
      </div>
      <div
        class="col-12 col-md-4 bg-light py-3"
      >
        <h1>
          Comfort
        </h1>
        <p
          class="lead"
        >
          Pamper your feline friend with the luxurious comfort of our scratching posts. Covered in soft, plush fabric, they offer a cozy retreat for your cat to relax and unwind.
        </p>
      </div>
      <div
        class="col-12 col-md-4 bg-light py-3"
      >
        <h1>
          Design
        </h1>
        <p
          class="lead"
        >
          Engage your cat's natural instincts and keep them entertained for hours with our interactive scratching posts. Featuring built-in toys and enticing textures, they stimulate your cat's senses and encourage active play.
        </p>
      </div>
    </div>
    <div
      class="row mb-4"
    >
      <div
        class="col-12py-3"
      >
        <p
          class="fs-1"
        >
          Empower Your Coding Journey with Every Scratch – Get Your Paws on Our Purr-fect Scratchers Today!
        </p>
      </div>
    </div>
  </div>
</div>
`;
